python/pyspark/sql/readwriter.py

```diff
@@ -828,8 +828,7 @@ def csv(self, path, mode=None, compression=None, sep=None, quote=None, escape=No
                     set, it uses the default value, ``,``.
         :param quote: sets the single character used for escaping quoted values where the
                       separator can be part of the value. If None is set, it uses the default
-                      value, ``"``. If you would like to turn off quotations, you need to set an
-                      empty string.
+                      value, ``"``. If empty string is set, it uses ``u0000``.
```

Comment:
`If empty string` -> `If an empty string `

Reply:
and maybe small additional comments for ``u0000`` would be nicer, e.g., it's null character.

Reply:
Fixing...